cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -230,6 +230,70 @@ def single_qubit_randomized_benchmarking(
     return RandomizedBenchMarkResult(num_clifford_range, gnd_probs)
 
 
+def parallel_single_qubit_randomized_benchmarking(
+    sampler: 'cirq.Sampler',
+    use_xy_basis: bool = True,
+    *,
+    qubits: Optional[Iterator['cirq.GridQubit']] = None,
+    num_clifford_range: Sequence[int] = [5, 18, 70, 265, 1000],
+    num_circuits: int = 10,
+    repetitions: int = 600,
+) -> dict:
+    """Clifford-based randomized benchmarking (RB) single qubits in parallel.
+
+    This is the same as `single_qubit_randomized_benchmarking` except on all
+    of the specified qubits in parallel, i.e. with the individual randomized
+    benchmarking circuits zipped together.
+
+    Args:
+        sampler: The quantum engine or simulator to run the circuits.
+        use_xy_basis: Determines if the Clifford gates are built with x and y
+            rotations (True) or x and z rotations (False).
+        qubits: The qubits to benchmark. If None, benchmark all of the qubits.
+        num_clifford_range: The different numbers of Cliffords in the RB study.
+        num_circuits: The number of random circuits generated for each
+            number of Cliffords.
+        repetitions: The number of repetitions of each circuit.
+
+    Returns:
+        A dictionary from qubits to RandomizedBenchMarkResult objects.
+    """
+
+    if qubits is None:  # pragma: no cover
+        try:
+            device = sampler.processor.get_device()  # type: ignore
```

Comment:
What's going on with the "type: ignore" commands?  You can say "device: TYPE = ..." in the first declaration to denote what type it should be, if the two device types disagree.

Reply:
I'll remove this code block since I would have to import `cirq_google` to get it typed properly.